src/ConnectedMode.UnitTests/QualityProfiles/QualityProfileDownloaderTests.cs

```diff
@@ -255,8 +254,8 @@ public async Task UpdateAsync_SavesConfiguration()
                 sonarQubeService ?? Mock.Of<ISonarQubeService>(),
                 bindingConfigProvider ?? Mock.Of<IBindingConfigProvider>(),
                 configurationPersister ?? new DummyConfigPersister(),
-                solutionBindingOperation ?? Mock.Of<ISolutionBindingOperation>(),
                 logger ?? new TestLogger(logToConsole: true),
+                solutionBindingOperation ?? Mock.Of<ISolutionBindingOperation>(),
```

Comment:
Re-ordered the parameters: the parameters in the public constructor come first, with the extra ones that are only set in the internal constructor set last.

The `ISolutionBindingOperation` is now a non-public parameter so it is set at the end.